tests/integration/test_using_cached_dependencies.py

```diff
@@ -120,7 +120,9 @@ def teardown_method(self, method):
                 self.branch, self.cloned_main_repo, self.main_repo_origin, [self.main_repo_commit]
             )
 
-    @pytest.mark.parametrize("env_name", ["pip_cached_deps", "gomod_cached_deps"])
+    @pytest.mark.parametrize(
+        "env_name", ["pip_cached_deps", "gomod_cached_deps", "npm_cached_deps", "yarn_cached_deps"]
```

Comment:
While testing it locally it works perfectly fine for me. If I use remote Cachito API, the response contains empty `packages` and `dependencies`.  Could you please check it with remote Cachito API?

Reply:
NVM, it was not related problem!